addon/controllers/fd-data-types-map.js

```diff
@@ -227,7 +211,7 @@ export default Ember.Controller.extend({
     @method serializeTypeMap
     @return {Boolean} Changed or not a combined type map.
   */
-  serializeTypeMap() {
+  findUnsavedFormData() {
```

Comment:
The method has been renamed, but nothing has changed inside, it seems to me strange.